client/src/modules/tags/templates/name.cell.html

```diff
@@ -0,0 +1,6 @@
+<div
+  class="ui-grid-cell-contents"
+  data-row="{{row.entity.name}}">
```

Comment:
Do you need the `data-row` binding?  I can't tell where it is used.

Reply:
It was used for `GridRow` tool for e2e tests, it is used also with: `/modules/roles/templates/action.cell.html`

Reply:
`GridRow` is used for clicking on dropdown menu items in the `actions` cells of tests.  What do you need to the `data-row` selector for here?  What are you checking/clicking on?

Reply:
I actually think this is breaking the end to end tests.  When I remove it, I think the end to end tests pass. 👍